react-frontend/src/components/FooterPages/privacy.js

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import DocumentTitle from 'react-document-title';
+
+import { PageContainer, GlobaStyleSize } from '../StyleComponents/pageContent';
+import { FAQHeading, Heading, Title } from '../StyleComponents/headings';
+import { HrefLink } from '../StyleComponents/htmlTags';
+
+function Privacy() {
+  return (
+    <DocumentTitle title="Privacy - Digital Government - Province of British Columbia">
+      <PageContainer>
+        <GlobaStyleSize />
+        <Title>Privacy</Title>
+
+        <Heading>Introduction</Heading>
+        <p>
+          The British Columbia (B.C.) government is committed to protecting your
+          privacy. The B.C. Government collects, uses and discloses your
+          personal information in accordance with the{' '}
+          <HrefLink href="https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96165_00">
+            Freedom of Information and Protection of Privacy Act
+          </HrefLink>{' '}
+          (<em>FOIPPA</em>) and other applicable legislation. ‘Personal
+          information’ is defined broadly in the Freedom of Information and
+          Protection of Privacy Act as recorded information about an
+          identifiable individual, other than contact information, which is the
+          information used to contact a person at a place of business. The
+          purpose of this privacy statement is to inform you of the personal
+          information that may be collected from you when visiting a B.C.
+          government website and how that information may be used.
+        </p>
+
+        <Heading>Scope</Heading>
+        <p>
+          This privacy statement relates only to the information automatically
+          collected from you as a result of your visit to the website. It does
+          not include information a website requests from you. Any additional
+          collections of personal information by a website will be addressed in
+          a collection notice on that website.
+        </p>
+
+        <FAQHeading>How is my personal information collected?</FAQHeading>
```

Comment:
If this is the same level of heading as the scope heading, they should probably be the same font size.  If it's meant to be a sub heading then I believe it should be an h3 not an h2

Reply:
I have changed `FAQHeading` to be a styled.h3 -- as for font-size, this was a deliberate choice: I want the FAQ headings to be presented by a screen reader as headings on the page (so that it's more intuitive that the paragraph following each FAQ Heading answers that heading's question), but style-wise, I want the text size to be small because the FAQ headings contain a lot of words and will look cumbersome if presented at that heading's normal font size.